yew/src/virtual_dom/vtag.rs

```diff
@@ -821,38 +822,41 @@ mod tests {
 
     #[test]
     fn filter_empty_string_classes() {
-        let a = html! { <div class=vec![""]></div> };
-        let b = html! { <div class=("", "")></div> };
+        let a = html! { <div class=classes!(vec![""])></div> };
+        let b = html! { <div class=classes!("", "")></div> };
         let c = html! { <div class=""></div> };
         let d_arr = [""];
-        let d = html! { <div class=&d_arr[..]></div> };
+        let d = html! { <div class=classes!(&d_arr[..])></div> };
 
-        macro_rules! has_class {
+        macro_rules! get_class {
             ($vtag:expr) => {
-                $vtag.attributes.iter().any(|(k, _)| k == "class")
+                $vtag
+                    .attributes
+                    .iter()
+                    .find_map(|(k, v)| if k == "class" { Some(v) } else { None })
             };
         }
 
         if let VNode::VTag(vtag) = a {
-            assert!(!has_class!(vtag));
+            assert_eq!(get_class!(vtag), Some(""));
```

Comment:
This test has been inverted because an empty `Classes` will always generated an empty `String`